eng/official-build.yml

```diff
@@ -2,6 +2,9 @@ trigger:
   branches:
     include:
     - main
+    - main-preview
+    - main-v2
+    - main-v3
     - release/v4.x
     - release/v3.x
     - release/v2.x

```

Comment:
@castrodd , I am keeping the release pipelines in triggers for now, we will need to update/remove these when we are ready to completely clean them up.

Reply:
Sounds good. Thank you!